.github/workflows/test.yml

```diff
@@ -50,6 +51,9 @@ jobs:
           - go-version: 1.15.x
             os: windows-latest
             go-flags: "-trimpath"
+          - go-version: 1.16.x
+            os: windows-latest
+            go-flags: "-trimpath"

```

Comment:
Please add the other two OSes (ubuntu and macos) to this list for Go 1.16.x.

Reply:
Done